### PR TITLE
Adds GitHub Teacher bio file

### PR DESCRIPTION
**Pull request by githubteacher**
:white_check_mark: Ready for Review :white_check_mark: 
- [x] Still needs proper descriptions and items inside of the `githubteacher.md` file.

This is related to work in #3

### CODE REVIEW

**Comment by githubteacher** [comment]
@githubstudent @shikharateverest @jhunnewell Could you all keep your eyes on this? It's not quite ready yet, but it will be soon. I'll let you know once i need more review.

**Comment by jhunnewell** [comment]
@githubteacher I will make sure this goes well. Your friend, @jhunnewell

**Comment by githubteacher** [comment]
@jhunnewell this is ready to go, can you just double check the "Files Changed" Tab to check for misspellings?

File: githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# GitHub Teacher Bio
+
+- Name: GitHub Sample Teacher
+- Occupation: Teacher
+- Home: San Francisco, CA, USA
+- Favorite Restaurant: Chez Maman
```
Comment (githubteacher) [inline githubteacher.md:6]: This is not to be confused with Chez Papa, which is next door

**Pull request merged**